app/instance-initializers/bugsnag.js

```diff
@@ -22,6 +23,12 @@ export default {
       Ember.onerror = function(error) {
         Bugsnag.context = getContext(router);
         const metaData = getMetaData(error, owner);
+        const user = getUser(owner);
+
+        if (user && typeof user === 'object') {
```

Comment:
Presumably the only time when this can happen is if the local version of `getUser` returns `null`.